### DISCUSSION TITLE
Grapes / Olives - Skip a row

### DISCUSSION THREAD

**Discussion by HC1Gunner** [Courseplay/Courseplay_FS22#1792] [Q&A] [answered]
For the most part using CP to mind grape or olive rows works fairly well. But when it comes harvest time the New Holland harvester has very hard time making U-turn for the next row. The vehicle has a very tight turn radius which CP does not take advantage of, instead CP has it swing out doing a large loop like it was a tractor pulling an implement.
So, what works well is to use the option to Skip a Row, but the problem with is, unlike doing regular fields with a tractor or combine, one it reaches the far side, it does not go back and do the even number rows. No matter what settings you generate a course with, it always starts and ends with the outside (1st) row, leaving you no choice but to do the even rows (skipped) manually yourself. Just wondering why this is not coding to behave the same way as a tractor or combine that skips rows, then reverse course to do the skipped rows?

v7.1.0.1

**Comment by BalticHawk** [comment]
The "Skip a row" option for normal field work and for grape and olive rows are two different settings and cannot be compared.
If you're using a sprayer for grapes, it works left and right, so you'll have to skip a row or you'll be working double. So it wouldn’t make sense to come back and do the skiped rows.

We all now know that the harvesters have a very small turning radius, but the CP developers struggle with giants physics that play ping-pong with the harvester when it's not absolutely straight. This is the reason for the large turning radius.

**Comment by HC1Gunner** [comment]
You need an alternate game plan when it comes to the grape /
olive harvester, these map makers that are creating maps with grapes and
olives already placed on a field or land, and next to no room to turn
around at the end.
My newest map (Italia Emilia Ext) is a nightmare, you can barely harvest
the grapes manually. Most of the fields CP can't be used, but if you could
devise another option like regular equipment where skipped rows would be
done on the return trip, then this would solve a lot of issues with tight
vineyards.

**Comment by Tensuko** [comment, marked as answer]
You may create a course, save it and see what you can do with the editor.
For some reasons (what we figured out during testing) it have to stay like that untill we find time to figure out something better.
It is not like we just have to change something here and there and its working...
